src/script/view_model/content/GroupCreationViewModel.ts

```diff
@@ -170,15 +170,16 @@ export class GroupCreationViewModel {
   readonly onGroupNameCancel = () => {
     this.groupName('');
   };
-  readonly onGroupNameChange = (event: any) => {
+
+  readonly onGroupNameChange = (event: Event) => {
     event.preventDefault();
-    const {value} = event.target;
+    const {value} = event.target as HTMLInputElement;
 
     const trimmedNameInput = value.trim();
-    const nameTooLong = trimmedNameInput.length > this.maxNameLength;
+    const nameTooLong = value.length > this.maxNameLength;
```

Comment:
Here it is safe to check the trimmed name. 
If the person names the group "`group                    `" We don't want to show the error since the extra spaces are going to be trimmed on `blur`